### PR TITLE
Added mixer sizes config param

### PR DESCRIPTION
**Pull request by filiplazovic**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -287,6 +288,12 @@ parameter_types! {
 	pub const MixerModuleId: ModuleId = ModuleId(*b"py/mixer");
 	pub const MinimumDepositLength: BlockNumber = 10 * 60 * 24 * 28;
 	pub const DefaultAdminKey: AccountId32 = AccountId32::new([0; 32]);
+		pub MixerSizes: Vec<Balance> = [
```
Comment (drewstone) [inline runtime/src/lib.rs:291]: spacing here is off

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -46,12 +46,13 @@ pub use sp_runtime::BuildStorage;
 pub mod currency {
 	use super::Balance;
 
-	pub const MILLICENTS: Balance = 1_000_000_000;
-	pub const CENTS: Balance = 1_000 * MILLICENTS; // assume this is worth about a cent.
-	pub const DOLLARS: Balance = 100 * CENTS;
+	// 12 zeros for easier testing
+	pub const DOLLAR: Balance = 1_000_000_000_000;
```
Comment (drewstone) [inline runtime/src/lib.rs:50]: I would still use the plural language as that's pretty standard around repos.

**Comment by filiplazovic** [comment]
Testing out with `webb-cli`

**Review by drewstone — CHANGES_REQUESTED** [review]
Small changes, looks good, but are there tests that we can run to ensure this works for edge cases or doesn't?

- What happens when I provide 0 mixer sizes?
- What happens if I provide mixer sizes of the same size?

Should we handle these cases directly?

**Comment by filiplazovic** [comment]
@drewstone I think there is no need since these are hard-coded config params. We are also not having checks for `DefaultAdmin` or `MaxTreeDepth`

**Comment by filiplazovic** [comment]
One more change to the runtime params and this can be merged

**Comment by drewstone** [comment]
Will also need to resolve conflicts @filiplazovic .

**Comment by filiplazovic** [comment]
@drewstone This can be merged. Weights should be the same, checking now...
